src/commands/appSettings/connectionSettings/sqlDatabase/azure/SqlDatabaseCreateStep.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type Database, type SqlManagementClient } from '@azure/arm-sql';
+import { parseAzureResourceId } from '@microsoft/vscode-azext-azureutils';
+import { AzureWizardExecuteStep, nonNullProp, nonNullValueAndProp } from '@microsoft/vscode-azext-utils';
+import { type Progress } from 'vscode';
+import { ext } from '../../../../../extensionVariables';
+import { localize } from '../../../../../localize';
+import { createSqlClient } from '../../../../../utils/azureClients';
+import { type ISqlDatabaseAzureConnectionWizardContext } from '../ISqlDatabaseConnectionWizardContext';
+
+export class SqlDatabaseCreateStep<T extends ISqlDatabaseAzureConnectionWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 210;
+
+    public async execute(context: T, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
+        progress.report({ message: localize('creatingDatabase', 'Creating new SQL database...') });
+
+        const client: SqlManagementClient = await createSqlClient(context);
+
+        const parsedServer = parseAzureResourceId(nonNullValueAndProp(context.sqlServer, 'id'))
+        const newDatabaseName: string = nonNullProp(context, 'newSqlDatabaseName');
+
+        const dbParams: Database = {
+            location: nonNullValueAndProp(context.sqlServer, 'location'),
+            sku: {
+                name: 'GP_S_Gen5',
+                tier: 'GeneralPurpose',
+                family: 'Gen5',
+                capacity: 1
+            },
+        };
+        context.sqlDatabase = await client.databases.beginCreateOrUpdateAndWait(parsedServer.resourceGroup, parsedServer.resourceName, newDatabaseName, dbParams);
+        ext.outputChannel.appendLog(localize('createdDatabase', 'Successfully created new SQL database "{0}"', context.sqlDatabase.name));
+
+        // Todo: This should be its own step
```

Comment:
I'll make this its own step once we add activity log children support, otherwise it doesn't really matter for now.